docker/pypi/dmwm-base/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM registry.cern.ch/cmsweb/cmsweb-base as cmsweb-base
-FROM registry.cern.ch/cmsweb/exporters as exporters
-FROM python:3.8-bullseye
+FROM registry.cern.ch/cmsweb/cmsweb-alma9-base:20250506-stable as cmsweb-base
```

Comment:
Dario, this image is only used to copy grid certificates from the image to new one. But as far as I know the CMSWEB managed to build k8s cluster where certificates are supplied by core node. If this is the case, we don't need `cmsweb-base` image (and copies of certs from it) at all. I would suggest to double check this with @arooshap and decide if we need to manage certificates on our own or if they are provided by k8s nodes we can just mount them within deployment.

Reply:
Apparently yeah we should be mounting them from the host [1] and use them to override what is inside the image [2]. I will have a look

[1] https://github.com/dmwm/CMSKubernetes/blob/9185d15c0c6f8a0408d243bd7fbac8a52212498a/kubernetes/cmsweb/services/reqmgr2.yaml#L138
[2] https://github.com/dmwm/CMSKubernetes/blob/9185d15c0c6f8a0408d243bd7fbac8a52212498a/kubernetes/cmsweb/services/reqmgr2.yaml#L52

Reply:
done